test/schema/export_test.js

```diff
@@ -21,14 +24,27 @@ describe('#exportToSchemasUniqueCases()', () => {
     let schemas = exportToSchemas([ns]);
     expect(schemas).to.have.length(16);
     expect(schemas).to.eql(expectedSchemas);
+    expect(validateSchema(schemas));
   });
 });
 
 function exportNamespaces(...namespace) {
   let schemas = exportToSchemas(namespace);
+  expect(validateSchema(schemas));
   return schemas;
 }
 
 function importFixture(name, ext='.json') {
   return JSON.parse(fs.readFileSync(`${__dirname}/fixtures/${name}${ext}`, 'utf8'));
 }
+
+function validateSchema(schema) {
+    const v = new validator();
+    const extendedSchema = importSchema();
+    return v.validate(schema, extendedSchema);
```

Comment:
Actually, I double-checked this. My first understanding is still the correct one. The first arg is the json to be validated against the second arg, the schema. In the example you posted, 4 is an attribute that gets checked for being a number or not. See the following other example from the same doc:

`validator.attributes.contains = function validateContains(instance, schema, options, ctx) {
  if(typeof instance!='string') return;
  if(typeof schema.contains!='string') throw new jsonschema.SchemaError('"contains" expects a string', schema);
  if(instance.indexOf(schema.contains)<0){
    return 'does not contain the string ' + JSON.stringify(schema.contains);
  }
}
var result = validator.validate("i am an instance", { type:"string", contains: "i am" });
// result.valid === true;`